src/pages/_index/_components/webassembly-components/index.tsx

```diff
@@ -0,0 +1,38 @@
+import { Section } from '@site/src/pages/_components/section';
+import { SectionHeading } from '@site/src/pages/_components/section-heading';
+import { SectionTag } from '@site/src/pages/_components/section-tag';
+import React from 'react';
+import styles from './webassembly-components.module.css';
+
+type Props = {};
+
+function WebAssemblyComponents({}: Props) {
+  return (
+    <Section color="light-gray" id="webassembly-components">
+      <div className="container">
+        <div className={styles.content}>
+          <div>
+            <SectionTag>WebAssembly Components</SectionTag>
+            <SectionHeading>New to Components?</SectionHeading>
+            <p>
+              WebAssembly components are a fundamentally finer-grained abstraction for application
+              composition and need a Wasm-specific orchestrator to take best advantage of them.
+            </p>
+            <p>
+              <a href="">Read our components starter guide</a>
+            </p>
+          </div>
+          <div>
+            <img
+              className={styles.image}
+              src="/img/pages/home/webassembly-components/components.svg"
+              alt=""
+            />
```

Comment:
This section's text is repeated, I honestly think a link is enough here though. The link pulls me back to the top of the page, should it go to the docs?

If a link isn't enough, the description at the top of https://deploy-preview-506--dreamy-golick-5f201e.netlify.app/docs/concepts/components#overview is nice.
```
Components are portable, interoperable WebAssembly binaries (.wasm files) that implement stateless logic. In wasmCloud, components typically enact the core logic of an application (for example, the API for a web application), while leaving abstracted, reusable capabilities such as key-value storage to the platform.
```

Reply:
Modified slightly